### DISCUSSION TITLE
物件偵測神器 : Yolov5

### DISCUSSION THREAD

**Discussion by handsomeniko** [1101-GS3073-C/Python-Community-GS3073-C#219] [General]
yolov5為作物件偵測分常實用的工具，我們第10組的靈籤辨識系統就是採用yolov5進行AI模型訓練
有興趣可以去研究看看:
https://pytorch.org/hub/ultralytics_yolov5/

**Comment by Shiba010** [comment]
在影像辨識上，yolo v5真的非常多人拿來應用！

**Comment by ayakasanma** [comment]
推推，廟公機器人用這個做很厲害！強大的技術後盾👍

**Comment by lx3211** [comment]
若想要針對自己的資料，訓練出自己的 YOLO 模型，可以用LabelImg準備好標註的圖片

**Comment by 32844583** [comment]
YOLO在實時目標檢測方面表現出色，因為它只需要對輸入圖像進行一次卷積神經網絡運算，就可以識別出所有物體。它的速度快，且準確度也相當高。

**Comment by gnsJhenJie** [comment]
現在已經有台灣人發明的YOLOv7了，大家可以參考看看~ 
論文上看到的表現比v5好，但有聽別人說，他跑別的資料集，v7會收斂不起來，v5效果則是好的。
https://github.com/WongKinYiu/yolov7

**Comment by skyling0299** [comment]
好有趣的技術

**Comment by Ben6uo** [comment]
感覺這個東西很有趣 但是應用的地方有很多吧

**Comment by liangyinglly** [comment]
這真的很有趣，下載輸入指令也不難，大家可以玩玩看
